.github/workflows/schema.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
         run: yarn build
 
       - name: Run tests
-        run: yarn test:ci
+        run: yarn test
```

Comment:
`schema.*` files were excluded from my global find/replace when I merged `test` and `test:ci` 🐒